lib/mbaasRequest/mbaasRequest.js

```diff
@@ -317,6 +317,30 @@ function _buildAppRequestParams(params) {
   };
 }
 
+/**
+ * Build request params for a storage request to an Mbaas
+ * @param params
+ * @returns Object
+ * @private
+ */
+function _buildStorageRequestParams(params) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this '_buildStorageRequestParams' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)